etn-ccis/blui-react-workflows
# Migrate react-auth-shared into react workflows

Fixes BLUI-4085.


#### Changes proposed in this Pull Request:
- Move react-auth-shared code into the src folder of react workflows
- Update scripts and configuration to build the project without the use of the @brightlayer-ui/auth-shared package


#### To Test:
- `git clone -b feature/4085-migrate-auth-shared-code git@github.com:etn-ccis/blui-react-workflows.git`
- `cd blui-react-workflows/login-workflow`
- `yarn start:example`
- ensure everything runs as it used to

- It would be ideal to test this in a new application as well, not just in the already provided example page just to see if it is integrated easily. In order to do this you can generate a new project from the CLI and replace all of the `@brightlayer-ui/react-auth-shared` imports with imports from `@brightlayer-ui/react-auth-workflow`
